pod/perlfunc.pod

```diff
@@ -9252,6 +9254,42 @@ The transliteration operator.  Same as
 L<C<yE<sol>E<sol>E<sol>>|/yE<sol>E<sol>E<sol>>.  See
 L<perlop/"Quote-Like Operators">.
 
+=item trim EXPR
+X<trim> X<remove_whitespace> (Perl 7+)
```

Comment:
This paragraph should not have any additional text outside of the `X<>` index blocks.

Reply:
This is removed in a commit I just landed